tests/test_records.py

```diff
@@ -105,6 +105,50 @@ def test_set_qualities_none(self):
         seq.qualities = None
         assert seq.qualities is None
 
+    def test_set_id(self):
+        seq = SequenceRecord("name", "A", "=")
+        with pytest.raises(AttributeError):
+            seq.id = "Obi-Wan"
+
+    def test_set_comment(self):
+        seq = SequenceRecord("name", "A", "=")
+        with pytest.raises(AttributeError):
+            seq.comment = "Hello there!"
+
+    @pytest.mark.parametrize(
+        ["record", "expected"],
+        [
+            (SequenceRecord("name", "A", "="), None),
+            (SequenceRecord("name ", "A", "="), None),
+            (SequenceRecord("name  ", "A", "="), " "),
+            (SequenceRecord("name", "A", "="), None),
+            (SequenceRecord("AotC I hate sand!", "A", "="), "I hate sand!"),
```

Comment:
:smile:

Reply:
https://www.youtube.com/watch?v=C_yWiW7OxZw

Reply:
Thanks, I got the Star Wars reference, but didn’t know the song (and know it’ll be stuck in my head for the rest of the day ...)

Reply:
Well you could always listen to "Last jedi knight" by the same musician to get it out of your head again ;-).